### ISSUE TITLE
Make date formats consistent for `.github` embeds

### ISSUE THREAD

**Issue by minalike** [python-discord/sir-lancebot#1093]
## Description

- Some dates are mm/dd/yyyy and others are dd/mm/yyyy. Make this consistent one way or another. Perhaps use localized timestamps since we have that option now instead of using the embed footers.
- Nitpick: I think "`minalike`'s GitHub profile info" should instead read: "minalike's GitHub Profile" (no backticks, proper casing)
- Nitpick: The usage of a large thumbnail in the user embed versus a small icon for the repo embed is weird. I think I prefer the small icon.

<img width="724" alt="image" src="https://user-images.githubusercontent.com/75038675/186555457-840634d2-d41b-4eae-a951-247cc2518d69.png">

## Would you like to implement a fix?

- [ ] I'd like to implement the bug fix
- [x] Anyone can implement the bug fix

Comments and suggestions welcome.

**Comment by wookie184** [comment]
I think the first one (Account created at) is a localised embed timestamp (shows dd/mm/yyyy for me), whereas the second one isn't, as localized timestamps in embed footers are very limited. I don't really have a preference on how to fix this inconsistency.

I agree with your second point on changing the title wording/formatting.

I don't really mind the icons being different. I quite like the bigger icon on the user command.

**Comment by Ibrahim2750mi** [comment]
If the status gets approved can I be assigned this issue?

**Comment by wookie184** [comment]
> If the status gets approved can I be assigned this issue?

There's still not really a conclusion on what should be changed for each part (apart from the second part).

**Comment by Ibrahim2750mi** [comment]
@wookie184 For the first part I think these lines are concerned


+ https://github.com/python-discord/sir-lancebot/blob/main/bot/exts/utilities/githubinfo.py#L350-L351
+ https://github.com/python-discord/sir-lancebot/blob/main/bot/exts/utilities/githubinfo.py#L264

We could use either dd/mm/yyyy or mm/dd/yyyy for both.

Don't know about the third part, I think its better the way its now.

**Comment by minalike** [comment]
Yeah I think it would be nice to move them out of the footer so we can use localized timestamps.
